src/main/java/adris/altoclef/trackers/storage/InventorySubTracker.java

```diff
@@ -213,13 +217,14 @@ private static boolean shouldIgnoreSlotForContainer(Slot slot) {
         // Because we don't want the bot to think we "have" an item if it's in our output slot. Otherwise it will
         // softlock because it will assume we're all good (we got the item!) when in reality we need to grab that item.
         //
-        // We also don't want our bot to think we "have" an item if it's in our armor/crafting/shield slots. That's annoying to work with.
+        // We also don't want our bot to think we "have" an item if it's in our armor/crafting output/shield slots.
         if (slot instanceof CraftingTableSlot && slot.equals(CraftingTableSlot.OUTPUT_SLOT))
             return true;
         if (slot instanceof PlayerSlot) {
             // Ignore non-normal inventory slots
             int window = slot.getWindowSlot();
-            return window < 9 || window > 44;
+            return window == 0 || (window > 4 && window < 9) || window > 44; // true if not a crafting input slot, or inventory slot.
```

Comment:
ender eye fix

we should count items that are in the 2x2 grid since they are in our inventory